library/Garp/Model/Db/User.php

```diff
@@ -298,6 +298,10 @@ public function invalidateEmailAddress(Garp_Db_Table_Row $user, $updateOrInsert
         $validationTokenColumn = $authVars['token_column'];
         $emailValidColumn = $authVars['email_valid_column'];
 
+        if ($updateOrInsert == 'insert' && $user[$emailValidColumn]) {
```

Comment:
Kun je hier nog zorgen dat er geen warning komt doordat `$emailValidColumn` niet gevonden is? (bv. met `array_get()`)

En waarom eigenlijk `return false`? Ik zou `return true` doen, er is een positief resultaat zou ik zeggen?